ledger-core/v2/virtual/virtual.go

```diff
@@ -93,6 +95,11 @@ func (lr *Dispatcher) Init(ctx context.Context) error {
 	lr.Conveyor.AddDependency(lr.runnerAdapter)
 	lr.Conveyor.AddDependency(lr.messageSenderAdapter)
 
+	if lr.ObjectCatalog == nil {
```

Comment:
In what case won't It be equal to nil?

Reply:
It can be not nil if it will set before Dispatcher.Init